Library/Homebrew/formula.rb

```diff
@@ -1381,12 +1381,15 @@ def stage
       mkdir_p env_home
 
       old_home, ENV["HOME"] = ENV["HOME"], env_home
+      old_java_options = ENV["_JAVA_OPTIONS"]
+      ENV.append "_JAVA_OPTIONS", "-Duser.home=#{env_home}"
```

Comment:
Wouldn't appending end up passing something like:

```
_JAVA_OPTIONS = -Duser.home=/Users/Dave/JavaIsFun -Duser.home=#{env_home}
```
With `append`? Guess there's an easy enough way to find out.